tests/test_parsers.py

```diff
@@ -14,6 +14,8 @@
 # test_upload_chunk_parser_media_type: Check the UploadChunkParser is
 #    setup with the correct media type: 'application/offset+octet-stream'
 #
+
+
```

Comment:
My only very minor suggestion in relation to these changes is to perhaps add one newline below line 5 and remove the newlines added in 17/18, so that the comment appears directly above the class and the two blank lines required for PEP8 instead appear on lines 5 and 6.

Reply:
Ok, sure